### ISSUE TITLE
calc_latlons and plot_groundtracks not implemented ?

### ISSUE THREAD

**Issue by TimotheeSimon** [alfonsogonzalez/AWP#17]
Hi Alfonso,

Is the main branch up-to-date? It seems that the calc_latlons and plot_groundtracks methods referenced in the Spacecraft class are not implemented. 
Cheers,

Timothée

**Comment by alfonsogonzalez** [comment]
Hi Timothée,

You're right, the Spacecraft class shouldn't have those methods since I haven't added the corresponding functions to the orbit_calculations and plotting_tools libraries. I was planning on adding them when they came up in the Fundamentals of Orbital Mechanics series, but I can move it up in the priority list.

If you're curious, I've shown these functions already in the videos. This playlist contains those videos: https://www.youtube.com/playlist?list=PLOIRBaljOV8ghaN9XcC4ubv-QLPOQByYQ

I marked this with astrodynamics and rocket trajectories label, because I just finished updating the Rocket class this week to be able to incorporate any launch site on Earth to orbit using two stages, and the groundtracks of the launches are interesting. I'll also be posting a little sneak peak of how that is going sometime this weekend.

Thank you for bringing this up!
